packages/docs/docs/cli/external-fhir-servers.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 2
+tags: [integration]
+---
+
+# Connecting to External FHIR Servers
+
+:::caution
+CLI connections to external FHIR servers is in Beta. Please join us on [Discord](https://discord.gg/medplum) with questions or email [hello@medplum.com](mailto:hello@medplum.com).
+:::
+
+When building an application, you many need to query or write data to an external FHIR server as part of your application's workflow. For example:
+
+- When a new patient is created, see if that patient already exists in another server
+- When an encounter is finalized, synchronize the encounter documentation to another FHIR server
+- On a bi-weekly basis, pull new FHIR data from a partner using a Bulk FHIR API
+
+To enable these scenarios, you will need a `clientId` or `clientSecret` to access the system you want to connect to. Please note that different systems have different levels of functionality, and so the commands in the CLI are not guaranteed to work.
```

Comment:
This is not strictly true.  That is just one auth pattern.  You could say something like "The most straightforward authentication option is the OAuth client credentials flow using `clientId` and `clientSecret`."  or something like that